x-pack/plugin/sql/qa/src/main/resources/docs.csv-spec

```diff
@@ -998,6 +1004,55 @@ SELECT COUNT(DISTINCT hire_date) unique_hires, COUNT(hire_date) AS hires FROM em
 // end::aggCountDistinct
 ;
 
+firstWithOneArg
+schema::FIRST(first_name):s
+// tag::firstWithOneArg
+SELECT FIRST(first_name) FROM emp;
```

Comment:
I think the examples would be more meaningful along with a GROUP BY.

Reply:
Generally we don't have group by for min/max/avg/kurtosis etc. examples

Reply:
We have at least one query with GROUP BY for each agg - the idea is to underline that this is in fact an agg (which might not be obvious when using implicit grouping).